app/src/ui/changes/commit-message.tsx

```diff
@@ -253,7 +255,7 @@ export class CommitMessage extends React.Component<
     const trailers = this.getCoAuthorTrailers()
 
     const commitCreated = await this.props.onCreateCommit(
-      summary,
+      this.props.singleFileCommit ? this.props.placeholder : summary,
```

Comment:
Actually on the second thought, this will not work if the user actually wants to commit their own summary. Let me fix this.